service/k8sclient/internal/kubectl/kubectl.go

```diff
@@ -280,6 +280,7 @@ func run(ctx context.Context, kubectlCmd []string, args []string, stdin interfac
 	err := cmd.Run()
 	if err != nil {
 		if strings.Contains(errBuf.String(), "NotFound") {
+			l.Warn(errBuf.String())
```

Comment:
errBuf.String() uses several times - looks like it has earned an honor to be a separate variable :)